serverless.yml

```diff
@@ -201,6 +204,17 @@ stepFunctions:
         "Clean-up Cluster":
           Type: Task
           Resource: cleanup_cluster
+          Next: "Checking Drain"
+        "Checking Drain":
+          Type: Task
+          Resource: check_drain
+          Retry:
+            -
+              ErrorEquals:
+                - UndrainedInstancesException
+              IntervalSeconds: 10
+              MaxAttempts: 10
+              BackoffRate: 1
```

Comment:
It doesn't need a catch, but it does need the retry. I assume 100 seconds should be enough for it to complete.